### ISSUE TITLE
Upgrading connectors.yml config file

### ISSUE THREAD

**Issue by mark-cameron** [silverstripe/silverstripe-sqlite3#57]
When flushing I get an error:
```
Uncaught exception: 
Not quoting the scalar "%$PDOConnector" starting with the "%" indicator character is deprecated since Symfony 3.1 and will throw a ParseException in 4.0 on line 5. 
(errno: 16384) in /var/www/html/vendor/symfony/yaml/Inline.php at line #357
```

Seems that since Symphony 3.1 starting a line with `%` means the line should be put into quotes. This now throws an error with symphony 4:
https://symfony.com/blog/new-in-symfony-3-1-yaml-deprecations#deprecated-starting-scalars-with-characters

Not sure which of my packages is requiring symphony 4 or above but it is quite an easy fix to add the single/double quotes to future proof this package.

**Comment by robbieaverill** [comment]
You're right, it's a safe and easy fix to quote the strings with % at the start. Would you like to make a PR against the 2.0 branch?

**Comment by mark-cameron** [comment]
Sure thing, how do you handle external PR's. Should I fork the repo and create the PR there?

**Comment by robbieaverill** [comment]
Yes please. You can find a guide for contributing here: https://docs.silverstripe.org/en/4/contributing/code/#step-by-step-from-forking-to-sending-the-pull-request

**Comment by robbieaverill** [comment]
Fixed with #58, thanks @mark-cameron!
